jobmaster/dm/task_manager_test.go

```diff
@@ -307,7 +299,7 @@ func TestTaskManager(t *testing.T) {
 	// mock check by interval
```

Comment:
line 296: if **task1 paused unexpected**, I think the worker status has changed to error and Paused, and the expected task stage is still Running. But my understanding is wrong with your code 🤔

Reply:
There are only 4 status of worker(creating/online/offline/finished), it a worker paused by error, it's regarded as 'Online'